frontend/simple/views/containers/Chatroom/ConversationGreetings.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div class="c-greetings">
+    <avatar v-for="user in founders"
+      size="sm"
+      class="c-avatar"
+      :src="user.picture"
+      :alt="user.displayName || user.name"
+    />
+    <div>
+      <message-notification :text="text" />
+    </div>
+  </div>
+</template>
+<style lang="scss" scoped>
+@import "../../../assets/sass/theme/index";
+
+.c-greetings {
+  align-self: center;
+  text-align: center;
+  margin-bottom: $gi-spacer-lg;
+}
+
+.c-avatar {
+  margin-left: -$gi-spacer-xs;
+  margin-right: -$gi-spacer-xs;
+  box-shadow: 0 0 0 1px $body-background-color;
+}
+</style>
+<script>
+import { groupA, users } from './fakeStore.js'
+import MessageNotification from '../../components/Chatroom/MessageNotification.vue'
+import Avatar from '../../components/Avatar.vue'
+
+export default {
+  name: 'ConversationGreetings',
+  components: {
+    MessageNotification,
+    Avatar
+  },
+  computed: {
+    // REVIEW... I dont like this....
```

Comment:
Can you be specific about what you don't like about this? My understanding is that all of this will be rewritten anyway and that this is just dummy data.

Reply:
Well.. that was the problem, part of it is real data (`greetingsMap`), part of it isn't (ex: `founders`), so I changed it to reflect that.